README.md

```diff
@@ -47,6 +47,7 @@ The packages intended for end developer usage are within the [pkg/client](pkg/cl
 ## Testing
 - [Build](docs/test/build_instructions.md)
 - [BDD tests](docs/test/bdd_instructions.md)
+- [BDD tests in detached mode](docs/test/starting_tests_fixtures.md)
```

Comment:
Why not have this as part of the BDD tests markdown file?

Reply:
to separate detached mode instructions from normal bdd tests instruction. It's easier to read this way.. Should I merge instead?

Reply:
Or you could put this link into that file.

@rolsonquadras  thoughts?

Reply:
Agree with @troyronda ie putting link for the new file in BDD tests. I see it as a part of BDD tests.

Reply:
done